plutus-core/untyped-plutus-core/src/UntypedPlutusCore/Evaluation/Machine/Cek/Internal.hs

```diff
@@ -202,7 +202,7 @@ data CekValue uni fun =
                              -- discharged values and discharging is expensive, so we don't want to
                              -- do it unless we really have to. Making this field strict resulted
                              -- in a 3-4.5% slowdown at the time of writing.
-      (CekValEnv uni fun)    -- For discharging.
+      !(CekValEnv uni fun)   -- For discharging.
```

Comment:
Isn't this potentially unused? Hence why we can leave it lazy. Although I agree it should always come from a strict context so :shrug:

Reply:
Wait... what's it even doing there?